src/main.rs

```diff
@@ -0,0 +1,162 @@
+#[macro_use]
+extern crate quicli;
+extern crate dialoguer;
+extern crate git2;
+extern crate ident_case;
+extern crate indicatif;
+extern crate liquid;
+extern crate regex;
+extern crate walkdir;
+
+use dialoguer::Input;
+use git2::{Config as GitConfig, Repository as GitRepository, RepositoryInitOptions};
+use quicli::prelude::*;
+use std::{env, fs};
+use walkdir::WalkDir;
+
+#[derive(Debug, StructOpt)]
+struct Cli {
+    #[structopt(long = "git")]
+    git: String,
+    #[structopt(long = "name")]
+    name: Option<String>,
+}
+
+main!(|args: Cli| {
+    let name = match &args.name {
+        Some(ref n) => n.to_string(),
+        None => query_name()?,
+    };
+
+    let project_dir = env::current_dir()
+        .unwrap_or_else(|_e| ".".into())
```

Comment:
curious about this `into` here

Reply:
I use `into` when I don't know/care what the type is supposed to be, but I know how to get it. I think it's OsString here? Or maybe PathBuf? I can be  more explicit if you like